src/status_im/translations/ru.cljs

```diff
@@ -55,255 +71,294 @@
                                            :other "часы"}
    :datetime-day                          {:one   "день"
                                            :other "дни"}
-   :datetime-multiple                     "c"
    :datetime-ago                          "назад"
    :datetime-yesterday                    "вчера"
    :datetime-today                        "сегодня"
 
-   ;profile
+   ;;profile
    :profile                               "Профиль"
-   :report-user                           "ПОЖАЛОВАТЬСЯ НА ПОЛЬЗОВАТЕЛЯ"
+   :edit-profile                          "Редактировать профиль"
    :message                               "Сообщение"
-   :username                              "Имя пользователя"
    :not-specified                         "Не указано"
    :public-key                            "Публичный ключ"
    :phone-number                          "Номер телефона"
-   :email                                 "Электронная почта"
-   :profile-no-status                     "Нет статуса"
+   :update-status                         "Обновите свой статус..."
+   :add-a-status                          "Добавить статус..."
+   :status-prompt                         "Укажите свой статус. Использование #хэштеги поможет другим узнать о вас и рассказать о том, что у вас на уме"
    :add-to-contacts                       "Добавить в контакты"
-   :error-incorrect-name                  "Выберите другое имя"
-   :error-incorrect-email                 "Неправильная электронная почта"
+   :in-contacts                           "В контактах"
+   :remove-from-contacts                  "Удалить из контактов"
+   :start-conversation                    "Начать разговор"
+   :send-transaction                      "Отправить транзакцию"
+   :testnet-text                          "Вы находитесь в {{testnet}} Testnet. Не отправляйте реальный ETH или SNT на свой адрес"
+   :mainnet-text                          "Вы в Mainnet. Реальный ETH будет отправлен"
 
    ;;make_photo
    :image-source-title                    "Изображение профиля"
    :image-source-make-photo               "Сфотографировать"
    :image-source-gallery                  "Выбрать из галереи"
-   :image-source-cancel                   "Отмена"
 
    ;;sharing
    :sharing-copy-to-clipboard             "Скопировать"
    :sharing-share                         "Поделиться..."
    :sharing-cancel                        "Отмена"
 
-   ;sign-up
+   :browsing-title                        "Просматривать"
+   :browsing-browse                       "@просматривать"
```

Comment:
Do not translate @browse if it contains "@". This is the name of the command and should  be in English